src/Network/Receive/ServerType0.pm

```diff
@@ -2861,143 +2270,6 @@ sub character_equip {
 	message($msg, "list");
 }
 
-sub storage_password_request {
```

Comment:
storage_password_request was removed from ServerType0 and added to Receive, but it's still in Sakexe_0